spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/RuntimeTemplate.java

```diff
@@ -34,77 +38,99 @@
  * @author Mark Fisher
  * @author Christian Tzolov
  * @author Chris Bono
+ * @author Corneil du Plessis
  */
 public class RuntimeTemplate implements RuntimeOperations {
 
-	private final RestTemplate restTemplate;
```

Comment:
@corneil did anything change in this file? I can't discern an actual change from the file format.